pkg/helpers/helpers.go

```diff
@@ -193,11 +193,23 @@ func CleanUpStaticObject(
 	case *rbacv1.RoleBinding:
 		err = client.RbacV1().RoleBindings(t.Namespace).Delete(ctx, t.Name, metav1.DeleteOptions{})
 	case *apiextensionsv1.CustomResourceDefinition:
-		err = apiExtensionClient.ApiextensionsV1().CustomResourceDefinitions().Delete(ctx, t.Name, metav1.DeleteOptions{})
+		if apiExtensionClient == nil {
+			err = fmt.Errorf("apiExtensionClient is nil")
```

Comment:
just a confusion here. all the places where we call` helpers.ApplyDirectly` and` helpers.CleanUpStaticObject `have validate apiExtensionClient and apiRegistrationClient. 
which scenario the imput clients are nil?

Reply:
ignore this. I missed that  management cluster has no CRD and APIService resources.